### ISSUE TITLE
Could you publish the training log for 'Novel Fine-tuning'?

### ISSUE THREAD

**Issue by ZhiXianZ** [gaobb/DCFS#5]
Thanks for your project. We were unable to achieve the average results in the paper as per your experimental parameters and configurations, and would like to see the magnitude of the fluctuations in the results over multiple times against the training log files.

**Comment by gaobb** [comment]
Thanks for your attention to our work. 

Please refer to [our supplementary_material](https://openreview.net/attachment?id=dVXO3Orjmxk&name=supplementary_material) for complete results of FSOD/FSIS and gFSOD/gFSIS (Tables 7, 8, 9, 10 and 11). In our supplementary material, we report the average AP/AP50 metric with 95% confidence interval over 10 seeds for FSOD/FSIS and gFSOD/gFSIS. 

Furthermore, I would like to see the gap between your reproduced results and the reported ones in our paper. If you have any other questions, further discussion is welcome.
